7a - nim project create - for issue #862

JamieDawson opened the pull request:
Here is a document for `7a. nim project create`.

![7a nim project create](https://user-images.githubusercontent.com/16840579/101098021-1551e100-3577-11eb-9367-02aa60623a1f.png)

rabbah commented on docs/key-value-storage.md:
```diff
@@ -0,0 +1,133 @@
+
+In this visual example, you can store user info, such as their City, State, and Country, and apply the correct values to them. This allows you to keep track of data and request the data when needed.
+
+![](https://github.com/JamieDawson/docs/blob/10b-two-types-storage-kv/static/img/kv-store-redis-example.png)
```
this is referring to your fork. instead use a relative link `./docs/...` so that it works for anyone's fork.

rabbah reviewed (commented):
check references to images that are fully qualified and referring to your fork - please replace those with local/relative links.

JamieDawson commented:
Issues fixed in this document:

1) Deleted `key-value-storage.md` file. That file isn't suppose to be in here.
2) Fixed broken images and replaced them with local ones.
3) Attached document to siebar.
4) `git rebased` all commits to one.

Pending Re-review.

rabbah reviewed (commented):
The file  `static/img/7a-nim-project-create.png` is not used in this PR?

rabbah reviewed (commented):
Also "7a" has no meaning in this repo. Should use a different naming convention (here just drop it.

JamieDawson commented:
All changes applied.

JamieDawson commented:
File that needs to be checked:
Getting started --->  Creating your project

rabbah commented on sidebars.js:
```diff
@@ -32,9 +32,10 @@ module.exports = {
+		'Getting Started': ['getting-started', 'building', 'nim-creating-project'],
 		'Developing and Deploying Serverless APIs': [
 			'sample-project-walkthrough',
+			'stateful-api-examples',
```
```suggestion
```

rabbah commented:
@JamieDawson I merged this PR but this one is relevant to the discussion in #30 as the reference docs for projects is currently not linked to this "how to" but should be so that a reader can dig into the details they will eventually need.
